app/lib/submission_builder/documents/adv_ctc_irs1040.rb

```diff
@@ -14,38 +14,34 @@ def dependent_xml(xml, dependent)
         xml.DependentDetail do
           xml.DependentFirstNm dependent.first_name
           xml.DependentLastNm dependent.last_name
-          xml.DependentNameControlTxt person_name_control_type(dependent.last_name)
+          xml.DependentNameControlTxt person_name_control_type(dependent.first_name)
```

Comment:
Em and I realized during the test case scenario that the reason for the control text is so that across multiple forms you can identify the same person even if the name is provided in a slightly different format. For that reason, if def makes sense to have the "control" be the first name.